contributed_definitions/nyaml/NXfabrication.yaml

```diff
@@ -16,25 +16,24 @@ NXfabrication(NXobject):
         specific enough to resolve the version.
   identifier(NXidentifier):
```

Comment:
NXidentifier will be removed.

Please see changes to NXfabrication [here](https://github.com/nexusformat/definitions/pull/1414/files#diff-d056d00e466575bfe638979229142136d1651baaa65460754caf9fd4e33bdb44) and adjust your changes on top accordingly.